### ISSUE TITLE
"Thanks"

### ISSUE THREAD

**Issue by ibc** [jtwebman/openapi#1]
Hi, thanks a lot for keeping a useful NPM library name such as "openapi" for nothing.

**Comment by jtwebman** [comment]
Yep sorry this was before openapi was even a thing. It was called swagger then but next time if you want a not used repo just ask and I might have given it to you. Now I am giving it to the swagger team.
